test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -389,6 +389,42 @@ await CliTester.Run(new RunConfiguration
             }, _output);
         }
 
+        [Fact]
+        [Trait(TestingTraits.TraitName.Category, TestingTraits.TestCategory.FinalIntegration)]
+        public async Task start_dotnet8_inproc_with_specifying_runtime_e2e()
+        {
+            await CliTester.Run(new RunConfiguration
```

Comment:
I think we also need a slight adjustment to use the consolidated zip (which contains all the needed artifacts from other builds) to be used for these E2E test. Also, we don't need to run it through the Shell class. We can do `dotnet` task in that pipeline to run the test project. will follow up with you offline.

Reply:
yeah this adjustment will be done in the pipeline itself. Let me know if this looks ok